.isort.cfg

```diff
@@ -17,3 +17,4 @@ known_standard_library=
 known_third_party=
 	djrad
 	djrad_rest
+	allianceutils
```

Comment:
This doesn't actually seem to be used anywhere in the code -- was it a bug fix or an accidental commit?

Reply:
was causing CI to fail - has since been fixed in master